app/services/stripe_service/api/stripe_api_wrapper.rb

```diff
@@ -299,5 +299,41 @@ def delete_account(community:, account_id:)
         account.delete
       end
     end
+
+    def create_payment_intent(community:, seller_account_id:, payment_method_id:, amount:, currency:, fee:, description:, metadata:)
+      with_stripe_payment_config(community) do |payment_settings|
+        Stripe::PaymentIntent.create(
+          capture_method: 'manual',
+          payment_method: payment_method_id,
+          amount: amount,
+          currency: currency,
+          confirmation_method: 'manual',
```

Comment:
Looks like you chose to implement the manual confirmation flow, instead of automatic confirmation one. Was there some reason for that? I originally spec'ed this to use automatic confirmation. I guess either one can work, just wondering if you had some reason to change that.